pandas/_libs/tslibs/dtypes.pyx

```diff
@@ -165,7 +165,7 @@ class FreqGroup(Enum):
     FR_MS = c_FreqGroup.FR_MS
     FR_US = c_FreqGroup.FR_US
     FR_NS = c_FreqGroup.FR_NS
-    FR_UND = -c_FreqGroup.FR_UND  # undefined
+    FR_UND = c_FreqGroup.FR_UND  # undefined
```

Comment:
Is `FR_UND`  essentially a dummy object so the sign doesn't matter?

Reply:
mostly, but we could probably cook up a scenario where the mismatch between FreqGroup.FR_UND and c_FreqGroup.FR_UND messed something up